frontend/packages/aquascope-editor/src/lib.ts

```diff
@@ -195,13 +132,38 @@ let parseWithDelimiters = (
   return { type: "ok", code: outputCode.join(""), ranges };
 };
 
+let buildStepperConfig = (config: StepperConfig) => {
+  config.focusedCharPos = [];
+  config.focusedPaths = new Map();
+  return (toks: string[], range: Range): boolean => {
+    let hasTag = toks.length >= 5 && toks.slice(0, 5).join("") === "step:";
+    return (
+      !hasTag ||
+      (() => {
```

Comment:
I know it's gross that JS doesn't have better support for statements-in-expressions. But IIFE syntax ("immediately-invoked function expression") is pretty gross to look at, I try to avoid it if necessary. You could just do:

```js
if (!hasTag) return true;
...
```